fiftyone/core/service.py

```diff
@@ -74,7 +76,9 @@ def __init__(self):
             )
         )
         self.child = None
-        if not self._disabled:
+        self._is_running = False
+
+        if not self._disabled and start:
```

Comment:
I think `self._disabled` needs to be acknowledge in more than just `__init__` now.

Reply:
Ah thanks! Fixed